pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -543,6 +543,94 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryBasicUserRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("imagestreams", "images"),
+				},
+				{
+					Verbs:     sets.NewString("create"),
+					Resources: sets.NewString("projectrequests"),
+				},
+				{
+					Verbs:     sets.NewString("list"),
+					Resources: sets.NewString("projects"),
+				},
+				{
+					Verbs:     sets.NewString("list"),
+					Resources: sets.NewString("subjectaccessreview", "localsubjectaccessreview"),
+				},
+			},
+		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("create", "edit", "list", "update", "watch"),
```

Comment:
Is there such a thing as "edit"? I think you want "update" (you already have it), "patch", and "delete"